src/test/java/com/wikia/webdriver/testcases/mercurytests/old/LoginTestsOld.java

```diff
@@ -129,7 +129,7 @@ public void MercuryLoginTest_011_japaneseUserLogIn() {
 //        Configuration.getCredentials().userNameJapanese2));
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3ACommentedOutCodeLine)